### PR TITLE
45 fix toggle menu

### PR DESCRIPTION
**Pull request by gaeVG**
- The transition of the toggle button has been corrected (the icon was returning to its initial state without transition)
- The position of the button has been modified to display it on the left of the logo
- The menu container has been modified to be displayed on the whole width of the screen when activated on mobile; a transition has been added for a better reading.
- The different states of the menu (active/not active) have been corrected and should not be a problem anymore, both on mobile and desktop computers

https://user-images.githubusercontent.com/16835349/164456659-34b47c7b-7472-4f5c-b596-ae43fcaffd18.mp4

### CODE REVIEW

**Comment by Lauraillouz** [comment]
2 little remarks to fix: 
- could you center the logo when on mobile version, then? (PS: I really like how the toggle button is on the left! <3 )
- when you're logged in, the log out link isn't center, could you fix it please?
Capture d’écran 2022-04-21 à 15 34 55

**Comment by gaeVG** [comment]
All the position problems have normally been solved; if you have a better suggestion in terms of design I leave it to you because I didn't really have any ideas that I like 🤐

File: src/styles/css/style.css [hunk 1]
```diff
@@ -313,13 +313,20 @@ button {
   z-index: 999; }
   .refemi-navbar .logo {
     position: relative;
-    margin: 1rem; }
+    margin-top: 0.9em;
+    margin-left: 12vh; }
```
[the remark this answers is not in the document]
Comment (gaeVG) [reply src/styles/css/style.css]: Is it possible to change these spacing parameters depending on the view (desktop / mobile) with the bulma classes? I tried to find this in the documentation but I didn't find it, that's why I specified it in the class and its media query

**Comment by Lauraillouz** [comment]
> All the position problems have normally been solved; if you have a better suggestion in terms of design I leave it to you because I didn't really have any ideas that I like 🤐

Could you just change every unit you've used to rem, please? I can see there are still some em and vh, let's try and remain consistant :)

**Comment by gaeVG** [comment]
> Could you just change every unit you've used to rem, please? I can see there are still some em and vh, let's try and remain consistant :)

[`fix: inconsistency of units`](https://github.com/Refemi/refemi_front/pull/114/commits/2d029422c8741f56e172425aab4f79a19b83a851) That's exactly what I did, right? 😥

File: src/styles/sass/layout/_header.scss [hunk 2]
```diff
@@ -36,51 +47,69 @@
...
+    display: block;
     position: absolute;
-    top: 50px;
-    left: -50px;
-    border-radius: $border-radius-1;
-    transition: 0.5s;
+    top: 8rem;
+    left: -100%;
+    width: 100%;
+    height: 100vh;
```
Comment (Lauraillouz) [inline src/styles/sass/layout/_header.scss:84]: These for example are still not in rem

File: src/styles/sass/layout/_header.scss [hunk 3]
```diff
@@ -36,51 +47,69 @@
...
+    padding-top: 2em;
```
Comment (Lauraillouz) [inline src/styles/sass/layout/_header.scss:85]: em

File: src/styles/sass/layout/_header.scss [hunk 4]
```diff
@@ -36,51 +47,69 @@
...
+        transition: 0;
+        box-shadow: none;
+      }
     }
 
     .nav-item {
       margin: auto;
+
       .btn-nav {
         padding: 3vh 1vw;
```
Comment (Lauraillouz) [inline src/styles/sass/layout/_header.scss:106]: since we're here, we could update the existing :)

**Comment by Lauraillouz** [comment]
> Indeed you were right, I had completely forgotten those!

You can now resolve the conflicts due to the merge of my PR hahaha

**Comment by gaeVG** [comment]
> > Indeed you were right, I had completely forgotten those!
> 
> You can now resolve the conflicts due to the merge of my PR hahaha

Yes, I'm on it 🤯

**Comment by gaeVG** [comment]
> > Indeed you were right, I had completely forgotten those!
> 
> You can now resolve the conflicts due to the merge of my PR hahaha

I had to solve a problem with the header disappearing when the mobile menu was open, but normally everything should be fine!

**Pull request merged**